config/routes.rb

```diff
@@ -317,13 +317,13 @@
       get '/doc_auth' => 'doc_auth#index'
       get '/doc_auth/:step' => 'doc_auth#show', as: :doc_auth_step
       put '/doc_auth/:step' => 'doc_auth#update'
-      get '/doc_auth/link_sent/poll' => 'doc_auth#doc_capture_poll'
```

Comment:
during the 10min of the deploy this might fail (old boxes pointing users to load a URL that is no longer supported), should we keep the old URL and have it render to the new controller? The API response is the exact same right?

Otherwise, this is not a super heavily-trafficked endpoint so maybe it's fine

Reply:
>Yeah, we could keep the existing URL.

Updated in 8e644dd.